autoinstrumentation/nodejs/src/autoinstrumentation.ts

```diff
@@ -50,11 +50,35 @@ function getMetricReader() {
     }
 }
 
+function getView() {
+  const buckets = process.env.OTEL_METRICS_EXPLICIT_BUCKET_HISTOGRAM;
+  const defaultHistogramBuckets = [
+    0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10,
```

Comment:
Does this follow https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-server ?